packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/testRunner.ts

```diff
@@ -95,7 +95,7 @@ export class TestRunner {
       runTestsByPathArgs = [];
     }
     const testNamePatternArgs = testName
-      ? ['--testNamePattern', `"${escapeStrForRegex(testName)}"`]
+      ? ['--testNamePattern', `${escapeStrForRegex(testName)}`]
```

Comment:
There's a spelling typo on line 68.  Though it's in the original code, it would be nice to get that fixed in this PR.

Reply:
There appears to be an issue escaping parenthesis as well.  I created a sample LWC test file in my org, and added the following:

```
    // ****************************************
    it('Example 1', () => {
        expect(1).toBe(1);
    });

    it("Example 2", () => {
        expect(1).toBe(1);
    });

    it(`Example 3`, () => {
        expect(1).toBe(1);
    });


    // ****************************************
    it('Example 4 (aka \'four\')', () => {
        expect(1).toBe(1);
    });

    it("Example 5 (aka 'five')", () => {
        expect(1).toBe(1);
    });

    it(`Example 6 (aka 'six')`, () => {
        expect(1).toBe(1);
    });


    // ****************************************
    it('Example 7 (aka "seven")', () => {
        expect(1).toBe(1);
    });

    it("Example 8 (aka (\"eight\")", () => {
        expect(1).toBe(1);
    });

    it(`Example 9 (aka "nine")`, () => {
        expect(1).toBe(1);
    });
```

When ` 'Example 4 (aka \'four\')' ` is passed to`escapeStrForRegex(testName)`, it returns:

"Example 4 \\(aka 'four'\\)"

expected:
"Example 4 (aka 'four')"